.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,173 @@
+##########################################################################
+#
+# WARNING: This file was generated by update.php.
+#
+# You can find the relevant template in the `/templates` folder.
+#
+
+name: Build and publish images to Docker Hub
+
+on:
+  push:
+    branches: master
+  # Once weekly On Sundays at 00:00 UTC.
+  schedule:
+    - cron: '0 0 * * 0'
+
+env:
+  REGISTRY_USERNAME: garypendergast
+  REGISTRY_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  PACKAGE_REGISTRY_HOST:
```

Comment:
Should this be empty?

Reply:
For the Docker Hub workflow, yes. The login command defaults to using Docker Hub, so no value is needed.